### PR TITLE
master -> main

### PR DESCRIPTION
**Pull request by leios**
I think the only thing we needed to change was `deploy.yml`, but I took the opportunity to also remove old travis infrastructure

### CODE REVIEW

File: .github/workflows/publish_container.yml [hunk 1]
```diff
@@ -2,16 +2,16 @@ name: Publish Docker
...
   push:
     branches:
-      - master
+      - main
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@main
```
Comment (ShadowMitia) [inline .github/workflows/publish_container.yml:10]: I don't think you're supposed to change that one.
```suggestion
    - uses: actions/checkout@master
```

File: .github/workflows/publish_container.yml [hunk 2]
```diff
@@ -2,16 +2,16 @@ name: Publish Docker
...
     - name: Publish to Registry
-      uses: elgohr/Publish-Docker-Github-Action@master
+      uses: elgohr/Publish-Docker-Github-Action@main
```
Comment (ShadowMitia) [inline .github/workflows/publish_container.yml:12]: I don't think you're supposed to change that one.
```suggestion
      uses: elgohr/Publish-Docker-Github-Action@master
```

**Review by ShadowMitia — CHANGES_REQUESTED** [review]

File: contents/domain_coloring/domain_coloring.md.bak [hunk 3]
```diff
@@ -1,185 +0,0 @@
-# Domain coloring
-
```
Comment (ntindle) [inline contents/domain_coloring/domain_coloring.md.bak:2]: Contex on removal?

File: contents/flood_fill/flood_fill.md.bak [hunk 4]
```diff
@@ -1,258 +0,0 @@
-# Flood Fill
-
-Flood fill is a method that is surprisingly useful in a large number of different situations and keeps finding me wherever I go.
-When I was completing my PhD, I had an idea to track superfluid vortices by using flood fill as a way to help mask out unnecessary features of the simulation.
```
Comment (ntindle) [inline contents/flood_fill/flood_fill.md.bak:4]: Context on removal?

File: .github/workflows/publish_container.yml (hunk 1 above)
Comment (ntindle) [reply .github/workflows/publish_container.yml:10]: Agreed

File: .github/workflows/publish_container.yml (hunk 2 above)
Comment (ntindle) [reply .github/workflows/publish_container.yml:12]: Agreed

**Review by ntindle — COMMENTED** [review]
LGTM overall, keep in mind you will need to be quick about it running and we may need to merge another PR right after to trigger the deployments since when this is merged the main branch wont exist

File: contents/domain_coloring/domain_coloring.md.bak (hunk 3 above)
Comment (leios) [reply contents/domain_coloring/domain_coloring.md.bak:2]: There were 2 `*.bak` files that were not meant to be committed. It was due to a temporary state in `aspell` for spell-checking I think

**Comment by leios** [comment]
Ok, so to be clear, the best thing to do is:
1. Change the name to main
2. Merge this
3. update another PR to make sure the CI builds
4. If that branch works, then update all PRs?

**Comment by Amaras** [comment]
If the branch is renamed successfully, all open PRs targets will be updated automatically as well.

**Pull request merged**